ansible/roles/mailcow/molecule/default/molecule.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+driver:
+  name: vagrant
+  provider:
+    name: libvirt
+
+lint: |
+  set -e
+  yamllint .
+
+platforms:
+  - name: debian9-instance
```

Comment:
maybe we should call this `debian10-instance` just to avoid mistakes

Reply:
Good catch!
Line 14 should be: `generic/debian9`